.github/workflows/docker-image.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         image_tag: ${{ matrix.browser }}
         push_git_tag: ${{ github.event_name == 'push' || github.event.pull_request.merged }}
         context: ${{ matrix.browser }}
+        stages_image_name: "${{ github.repository_owner }}dev/ledgersmb_circleci-browsers-${{ matrix.browser }}-stages"
```

Comment:
That will force developers to get a ${{ github.repository_owner }}dev docker repository for them to test. Shouldn't we adjust for ledgersmbdev or ${{ github.repository_owner }} to simplify?

Reply:
It's not, all 3 mandates developers to set up a dev repository on docker hub.

Reply:
Ok. But: if it requires the developer to either create a namespace on docker hub by the name of `$developer_name` or `${developer_name}dev`, what's the difference? I mean: both require a Docker Hub repository?

Reply:
They need a Docker Hub repository in their own name too. All in all not a practical setup. But I think it's the best we can do for now.